data/items.js

```diff
@@ -3117,6 +3117,22 @@ let BattleItems = {
 		desc: "Evolves Gloom into Vileplume, Weepinbell into Victreebel, Exeggcute into Exeggutor or Alolan Exeggutor, Eevee into Leafeon, Nuzleaf into Shiftry, and Pansage into Simisage when used.",
 		shortDesc: "Evolves certain species of Pokemon when used.",
 	},
+	"leek": {
+		id: "leek",
+		name: "Leek",
+		fling: {
+			basePower: 60,
+		},
+		spritenum: 475, //TODO: Sprite, using Stick as a placeholder
```

Comment:
```suggestion
		spritenum: 475,
```

No placeholder, it's literally a Stick.

Reply:
Wasn't sure.

HoeenHero - Last Friday at 12:25 AM
Not sure if we have a sprite id for leek yet but is is different

Reply:
I might be misinterpreting HoeenHero, but he's wrong about that.